File: 01.js
const texto = prompt('Ingresa la palabra').toLowerCase().trim();
let texArr = texto.split('');
const vocales = ['a', 'e', 'i', 'o', 'u'];

function contarVocales(arr) {
  let contador = 0;
  for (let i = 0; i < arr.length; i++) {
    if (vocales.includes(arr[i])) {
      contador++;
    }
  }
  return contador;
}
const totalVocales = contarVocales(texArr);
console.log(`La palabra ${texto} tiene ${totalVocales} vocales`);

File: 01.py
print(f'Ingresa la palabra: ')
palabra = input().lower().strip()
vocales = ['a', 'e', 'i', 'o', 'u']


def contar_vocales(palabra):
    contador = 0
    for letra in palabra:
        if letra in vocales:
            contador += 1
    return contador


print(f'La palabra tiene {contar_vocales(palabra)} vocales')
